docs/Schema-changes.md

```diff
@@ -0,0 +1,176 @@
+### Changes to Aries VCR issuer schema
+
+The Aries VCR application is undergoing changes to enhance user experience on applications such as Orgbook. The primary change to the Aries VCR application is in the way that credential schemas are issued.  
+  
+Included below are a list of optional changes to credential schemas that issuers may be interested in making. For the purpose of clearly displaying how these changes are being used, we've provided examples of the credential changes in the new Orgbook site. You can find the code for the new Orgbook frontend [here](https://github.com/bcgov/orgbook-bc-client-vue).
+
```

Comment:
I suggest to also drop a link to the example issuer here:  https://github.com/bcgov/aries-vcr-issuer-controller/tree/master/issuer_controller/config